lib/diagnostic.js

```diff
@@ -28,7 +34,14 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
+let owner = [
+  {
+    name: 'Ken Nguyen',
+    address: '51 Welcher Street',
+    numberOfPets: 2
+  }
+];
+
```

Comment:
Why did you put your object inside of an array?  Do you understand object literals?  Let's talk this weekend or sometime early next week. If you need more help lets go over this next week.